src/Microsoft.CodeQuality.Analyzers/Core/Maintainability/ReviewUnusedParameters.cs

```diff
@@ -110,6 +110,12 @@ public override void Initialize(AnalysisContext context)
                         return;
                     }
 
+                    // Ignore property accessors.
+                    if (method.IsPropertyAccessor())
```

Comment:
Can you please add a unit test for this case in analyzer test file `ReviewUnusedParametersTests.cs`?

Reply:
Agree. Moving them from fixer tests.

---
In reply to: [161036487](https://github.com/dotnet/roslyn-analyzers/pull/1516#discussion_r161036487) [](ancestors = 161036487)